django/src/rdwatch/views/model_run.py

```diff
@@ -229,6 +231,13 @@ def get_queryset():
                     then=JSONObject(
                         proposed=Coalesce(Subquery(proposed_count_subquery), 0),
                         other=Coalesce(Subquery(other_count_subquery), 0),
+                        groundTruths=Subquery(
```

Comment:
```suggestion
                        ground_truths=Subquery(
```

For consistency with the case of other fields in this query.